src/InventoryManager.java

```diff
@@ -0,0 +1,263 @@
+                        System.out.println("Invalid item " + name);
+                        continue;
+                    }
+                    System.out.println("Enter new quantity: ");
+                    String quantity = br.readLine();
+                    if(!isInteger(quantity)) {
+                        System.out.println("Invalid Quantity " + quantity);
+                        continue;
+                    }
+                    Inventory inventory = itemMap.get(name);
+                    inventory.setQuantity(Integer.parseInt(quantity));
+                }
+
+                // update item price
+                else if(option.equals("5")) {
+                    if(itemMap.size() <= 0) {
+                        System.out.println("No items found.");
+                        continue;
+                    }
+                    System.out.println("Choose from items below by name");
+                    Iterator iter = itemMap.keySet().iterator();
+                    while(iter.hasNext()) {
+                        System.out.println((String)iter.next());
+                    }
+                    String name = br.readLine();
+                    if(!itemMap.containsKey(name)) {
+                        System.out.println("Invalid item " + name);
+                        continue;
+                    }
+                    System.out.println("Enter new price: ");
+                    String price = br.readLine();
+                    if(!isDouble(price)) {
+                        System.out.println("Invalid price " + price);
+                        continue;
+                    }
+                    Inventory inventory = itemMap.get(name);
+                    inventory.setPrice(Double.parseDouble(price));
+
+                }
+
+                // quit processing while loop
+                else if(option.equals("6")) {
+                    System.out.println("Quit");
+                }
+                else {
+                    System.out.println("value = " + option);
+                }
+            }
+        }
+        catch(Exception e ) {
+            e.printStackTrace();
+        }
+    }
+
+    // helper to validate String is an integer
+    private boolean isInteger(String option) {
+        boolean test = true;
+        try {
+            int i = Integer.parseInt(option);
+        }
+        catch(NumberFormatException nfe) {
+            test = false;
+        }
+        return test;
+    }
+
+    // helper method that validates String is a double
+    private boolean isDouble(String option) {
+        boolean test = true;
+        try {
+            double i = Double.parseDouble(option);
+        }
+        catch(NumberFormatException nfe) {
+            test = false;
+        }
+        return test;
+    }
+
+    // Create item method based on category passed in during create item option #1.
+    private Inventory createItem(String name, double price, int quantity, String category) {
```

Comment:
Per the assignment this should be moved to the `Inventory` class and made `static` so that the `InventoryManager` could all the `Inventory` class to create an object itself.